### PR TITLE
[US7 Task 53] added component for area chart

### PR DESCRIPTION
**Pull request by sverma89asu**
Screenshot 2024-02-15 at 7 19 19 PM

### CODE REVIEW

**Comment by sverma89asu** [comment]
Screenshot 2024-02-15 at 7 46 10 PM

**Comment by sverma89asu** [comment]
Screenshot 2024-02-15 at 8 46 37 PM

File: react-ui/src/areachart/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                <div style={{ width: '90%' }}>
+                    <h4 style={{ textAlign: 'center' }}>{title}</h4>
+                    <ResponsiveContainer height={600}>
+                        <AreaChart
+                            margin={{top: 20, right: 40, bottom: 70, left: 30}}
+                            data = {apiData}
+                        >
+                            <defs>
+                                <linearGradient id="colorExpected" x1="0" y1="0" x2="0" y2="1">
+                                <stop offset="5%" stopColor="#D04848" stopOpacity={0.8}/>
```
Comment (shahih25) [inline react-ui/src/areachart/index.js]: nit: format (add a tab before <stop> element)
`
<linearGradient id="colorExpected" x1="0" y1="0" x2="0" y2="1">
    <stop offset="5%" stopColor="#D04848" stopOpacity={0.8}/>
    <stop offset="95%" stopColor="#D04848" stopOpacity={0.1}/>
</linearGradient>
`

File: react-ui/src/areachart/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                                <stop offset="95%" stopColor="#D04848" stopOpacity={0.1}/>
+                                </linearGradient>
+                                <linearGradient id="colorActual" x1="0" y1="0" x2="0" y2="1">
+                                <stop offset="5%" stopColor="#6895D2" stopOpacity={0.8}/>
```
Comment (shahih25) [inline react-ui/src/areachart/index.js]: nit: same as above

**Review by shahih25 — APPROVED** [review]

**Pull request merged**